client/dashboard/data/site-staging-site.ts

```diff
@@ -14,3 +14,10 @@ export async function deleteStagingSite( stagingSiteId: number, productionSiteId
 		apiNamespace: 'wpcom/v2',
 	} );
 }
+
+export async function fetchStagingSiteData( productionSiteId: number ) {
```

Comment:
Nit: we don't usually use `Data` prefix; what about

```suggestion
export async function fetchStagingSiteOf( productionSiteId: number ) {
```

Reply:
I like it, updated it in 7a1ef49bbb061d092f46cd9e2b85583508b6a003